helm/ph-ee-engine/values.yaml

```diff
@@ -1945,11 +1945,12 @@ keycloak:
       value: /realm/kong-keycloak-realm.json     
 
 integration_test:
-  enabled: false
+  enabled: true
   replicas: 1
-  image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-integration-test"
-  imageTag: latest
-  imagePullPolicy: "Never"
+  image: docker.io/openmf/ph-ee-integration-test:latest
+  imagePullPolicy: Always
+  gradle:
+    command: ./gradlew test -Dcucumber.filter.tags="@gov"
```

Comment:
need to figure out how to test other tags ( eg; @amsintegration)

Suggestion: 
`./gradlew test -Dcucumber.filter.tags="@gov" | ./gradlew test -Dcucumber.filter.tags="@amsIntegration" | echo ""`